lib/_stream_transform.js

```diff
@@ -202,6 +203,32 @@ Transform.prototype._destroy = function(err, cb) {
   });
 };
 
+Transform.prototype.acquireStandardStream = function() {
+  emitExperimentalWarning('Transform.acquireStandardStream');
+  return new TransformStream({
+    start: (controller) => {
+      this.on('data', (chunk) => {
+        controller.enqueue(chunk);
+      });
+      this.on('end', () => controller.close());
+      this.once('error', (e) => controller.error(e));
```

Comment:
listening to the error event with `once`  is not correct. `'error'` _could_ be emitted multiple times by descendants.

Reply:
whatwg streams only error once